package.json

```diff
@@ -14,12 +14,16 @@
     "@babel/cli": "^7.2.3",
     "@babel/core": "^7.3.4",
     "@babel/preset-env": "^7.3.4",
-    "@babel/preset-react": "^7.0.0"
+    "@babel/preset-react": "^7.0.0",
+    "babel-cli": "^6.26.0"
   },
   "babel": {
     "presets": [
       "@babel/preset-env",
       "@babel/preset-react"
     ]
+  },
+  "dependencies": {
+    "redux-undo": "^0.6.1"
```

Comment:
What is this for?

Reply:
I think that's a dependency for a function I ended up not using. I didn't realize that got added. You can delete that.